src/xdocs/google_style.xml

```diff
@@ -2437,6 +2437,31 @@
           </table>
         </div>
       </subsection>
+      <subsection name="Overrides" id="Google_Overrides">
+        <p>
+          To change the default severity level, use the following custom property variable:
+          <br/>
+          <br/>
+          <code>org.checkstyle.google.severity=${SeverityLevel}</code>
+          <br/>
+          <br/>
+          where <code>SeverityLevel</code> can be any one of them:
+          <code>ignore</code>, <code>info</code>, <code>warning</code>, <code>error</code>.
+          Default is <code>Warning</code>.
+        </p>
+        <p>
+          Visit here for more details about Severity Levels:
+          <a href="https://checkstyle.sourceforge.io/property_types.html#SeverityLevel">
+            Severity Level
+          </a>
+        </p>
+        <p>
+          Visit here for more details about how to use custom properties:
```

Comment:
`...custom properties that can be defined by '-p <propertiesFile>'`

Reply:
I followed your this suggestion: https://github.com/checkstyle/checkstyle/pull/15072#discussion_r1648976315